### ISSUE TITLE
Ablation study? 

### ISSUE THREAD

**Issue by jukofyork** [kvcache-ai/ktransformers#5]
It would be interesting to see the effect of each optimisation separately.

Some of the ideas might be very easy to use in other inference engines and others much harder to implement, so it would be very helpful to get an idea of the gains for each feature separately.

I've linked this to the `llama.cpp` discussions:

https://github.com/ggerganov/llama.cpp/discussions/8721

As the "Arithmetic Intensity Guided Offloading" is clearly something they could do quite easily and likely give a significant boost to MoE models.

**Comment by james0zan** [comment]
Yes, we will present more results later. Actually, our vision for KTransformers is to serve as an experimental platform used to develop prototypes faster. The more mature ones can then be adopted into more popular inference engines like llama.cpp and vLLM.

**Comment by ErvinXie** [comment]
This issue has been automatically closed due to inactivity for 60 days.

If you believe this issue is still relevant, please feel free to reopen it with additional information or context.

Thank you for your contribution to KTransformers!

---
此 issue 因超过 60 天无活动而被自动关闭。

如果您认为此问题仍然存在，请随时重新打开并提供更多信息或上下文。

感谢您对 KTransformers 的贡献！
